packages/headless/src/database/mongodb/brand/brand.model.ts

```diff
@@ -0,0 +1,62 @@
+import { model, Schema } from 'mongoose';
+import * as crypto from 'crypto';
+
+import { Brand } from 'src/entity/brand';
+
+const BrandSchema = new Schema<Brand>({
+    id: {
+        type: String
+    },
+    info:{
+        name: {
+            type: String,
+            required: true,
+            unique: true
+        },
+        brandId:{
+            type: String,
+            default: crypto.randomUUID(),
```

Comment:
remove default from here. branId should be provided when being saved

Reply:
The save method is identified by only the mongodb in its seperate module, so keeping it in the schema isn't going to make big difference unless we do it in the service or repository layer i suppose.
Can you make it a bit more clear? What exactly should I do here